.github/workflows/deploy.yml

```diff
@@ -108,7 +108,7 @@ jobs:
           image: ${{ steps.push_image.outputs.ecr_image_location }}
           access-role-arn: arn:aws:iam::${{ secrets.AWS_ACCOUNT }}:role/${{ inputs.environment }}-funding-service-app-runner-build-role
           region: eu-west-2
-          port: 5000
+          port: 8080
```

Comment:
Probably want to update terraform to point at 8080 as well, just to keep everything in sync.